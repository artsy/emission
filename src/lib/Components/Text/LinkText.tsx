import styled from "styled-components/native"

export const LinkText = styled.Text`
  text-decoration-line: underline;
`
