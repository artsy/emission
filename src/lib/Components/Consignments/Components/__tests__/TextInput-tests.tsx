import "jest-snapshots-svg"
import * as React from "react"
import * as renderer from "react-test-renderer"
import Text from "../TextInput"

it("shows an activity indicator when searching ", () => {
  const component = renderer.create(<Text text={{ value: "My mocked" }} searching={true} />).toJSON()
  expect(component).toMatchSnapshot()
})

it("does not have an activity when searching ", () => {
  const component = renderer.create(<Text text={{ value: "My mocked" }} searching={true} />).toJSON()
  expect(component).toMatchSnapshot()
})
