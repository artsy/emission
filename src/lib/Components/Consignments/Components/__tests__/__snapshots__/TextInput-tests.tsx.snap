// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not have an activity when searching  1`] = `
<View
  style={
    Array [
      undefined,
      Object {
        "flex": 1,
        "maxHeight": 40,
      },
    ]
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "height": 40,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCorrect={false}
      clearButtonMode="while-editing"
      keyboardAppearance="dark"
      onBlur={[Function]}
      onFocus={[Function]}
      placeholderTextColor="#666666"
      selectionColor="#cccccc"
      style={
        Array [
          Object {
            "backgroundColor": "black",
            "color": "white",
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
            "fontFamily": "AGaramondPro-Regular",
            "fontSize": 20,
            "height": 40,
          },
          undefined,
        ]
      }
      underlineColorAndroid="transparent"
      value="My mocked"
    />
    <ActivityIndicator
      animating={true}
      color="#999999"
      hidesWhenStopped={true}
      size="small"
    />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#666666",
        "height": 1,
      }
    }
  />
</View>
`;

exports[`shows an activity indicator when searching  1`] = `
<View
  style={
    Array [
      undefined,
      Object {
        "flex": 1,
        "maxHeight": 40,
      },
    ]
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "height": 40,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCorrect={false}
      clearButtonMode="while-editing"
      keyboardAppearance="dark"
      onBlur={[Function]}
      onFocus={[Function]}
      placeholderTextColor="#666666"
      selectionColor="#cccccc"
      style={
        Array [
          Object {
            "backgroundColor": "black",
            "color": "white",
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
            "fontFamily": "AGaramondPro-Regular",
            "fontSize": 20,
            "height": 40,
          },
          undefined,
        ]
      }
      underlineColorAndroid="transparent"
      value="My mocked"
    />
    <ActivityIndicator
      animating={true}
      color="#999999"
      hidesWhenStopped={true}
      size="small"
    />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#666666",
        "height": 1,
      }
    }
  />
</View>
`;
